src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/DataSetFilesControllerTests.cs

```diff
@@ -1779,6 +1779,78 @@ public async Task ReleaseFileSummariesContainHtml_HtmlTagsAreStripped()
         }
     }
 
+    public class ListSitemapItemsTests(TestApplicationFactory testApp) : DataSetFilesControllerTests(testApp)
+    {
+        public override async Task InitializeAsync() => await TestApp.StartAzurite();
+
+        private async Task<HttpResponseMessage> InvokeListSitemapItems(
+            WebApplicationFactory<Startup>? app = null)
+        {
+            var client = (app ?? BuildApp()).CreateClient();
+
+            return await client.GetAsync("/api/data-set-files/sitemap-items");
+        }
+        
+        [Fact]
+        public async Task ListSitemapItems()
+        {
+            Publication publication = _fixture.DefaultPublication()
```

Comment:
Not super important, but I think the preference is to use `var` in variable assignments rather than the type.

Reply:
Aye good spot, I wasn't sure which way to go here because the rest of the file uses the explicit types. 

I chose consistency because I know that's something we're really keen on - I'm very happy to switch it over to var, but would you like me to switch over the rest of the extant tests in this file too?